app/routes/circulars/circulars.server.ts

```diff
@@ -213,7 +210,34 @@ export async function search({
       size: limit,
       track_total_hits: true,
     },
-  })
+  }
+
+  let searchResult
+  try {
+    searchResult = await client.search(searchBody)
+  } catch (error) {
+    console.error(
+      'Search with queryObj failed, falling back to multi_match',
+      error
+    )
```

Comment:
Although this is is a user error, it doesn't suggest anything wrong with our code, so we shouldn't log it.

```suggestion
```